pkg/cmd/pulumi/operations/up.go

```diff
@@ -816,6 +824,16 @@ func validatePolicyPackConfig(policyPackPaths []string, policyPackConfigPaths []
 	return nil
 }
 
+func validateAttachDebuggerFlag(args []string) error {
+	for _, arg := range args {
+		if arg != "program" && arg != "plugins" && arg != "all" && !strings.HasPrefix(arg, "plugin=") {
```

Comment:
"plugin=" rather than "plugin:"?

Reply:
Ah yes, I think it should be `plugin:`, but confused myself at some point.  That's probably less confusing when doing `--attach-debugger=plugin:bla` rather than `--attach-debugger=plugin=bla`.  Changed it now.